platform/src/test/kotlin/org/stellar/anchor/platform/service/Sep24MoreInfoUrlConstructorTest.kt

```diff
@@ -146,17 +134,6 @@ class Sep24MoreInfoUrlConstructorTest {
     assertEquals("some-wallet", claims["client_name"])
   }
 
-  @Test
```

Comment:
Is this test no longer relevant?

Reply:
Yeah this is no longer the case. Non custodial client needs to have at least one domain, or error will be thrown at config level.